go.sum

```diff
@@ -38,6 +38,8 @@ github.com/asaskevich/govalidator v0.0.0-20180720115003-f9ffefc3facf/go.mod h1:l
 github.com/asaskevich/govalidator v0.0.0-20190424111038-f61b66f89f4a/go.mod h1:lB+ZfQJz7igIIfQNfa7Ml4HSf2uFQQRzpGGRXenZAgY=
 github.com/aws-controllers-k8s/runtime v0.0.0-20210204203051-91a6bd53a48c h1:0hHYJ3fVDLSnENSFlYxrajjpFOehEnzYPiX1VIWB4Xw=
 github.com/aws-controllers-k8s/runtime v0.0.0-20210204203051-91a6bd53a48c/go.mod h1:ZENN/6/CjjeUNVEhli84fDgkxDw21zsm7xtzHuCJAGg=
```

Comment:
feel free to run `go mod tidy` at some point in the future which will remove the above lines.